meetings/2024-10-23.md

```diff
@@ -0,0 +1,154 @@
+ * RG: I've begun that process by putting what those generic requirements might look like in terms of classes of products.
+ * Elf: I've been doing webhooks in the last few months, if we are doing errors then it impacts the notification recievers as they need to check if at as error message etc if errors are included with notifications.
+
+### Activity Mapping for Notifications
+
+https://github.com/solid/notifications/issues/183
+
+* RG: Added a new column; need to revisit this. Looking at what are the container mappings, because this is something we need to revisit soon, the purpose of this issue was to have an [?? office hours STM??]
+* eP: Should not map to HTTP Headers. Solid or LWS should define set of operations and notification system hooks into set of operations. Then there will be HTTP interface that is binding this to HTTP methods but notif. system should not bind directly to that. Relevant to discuss in relation to access control and notif system, both should have a common layer of internal operations supported by the storage. That is something LWS may need to take care of by making those operations defined
+* TBL: Tricky thing about notifications is that people generalise. People say, e.g. don't just standardise a version of websockets or "there is a world of notification out there and we should model them all in our notifications". Can be unhelpful by running into danger of assuming that there will be lots of different notif features that will have too much in common, e.g., if user moves mouse into space you'll get a notif - lots of cases where the timescale of things are very different. I hope that we have a very effective & test notif. systems for what I need in SolidOS where we can update the display when someone changes something. Interested in relation to local-first and CRDTs, if you're making a CRDT you cant be ??? about it. I hope if we are trying to make something that works at CRDT level + at higher levels we don't end up causing confusion. Lets make sure that notifs at one level work well, and then generalise.
+* MDJ: Question about linked issue - it sounds like a better idea to just use what is there rather than taking to office hours or take to working group
+* RG: A lot is already implemented e.g. in CSS, and there is already usage. But in implementation people found underspecified cases and need clearer requirements based on those things. We need to take another specification step to iron out those details.
```

Comment:
```suggestion
* MDJ: Question about linked issue — it sounds like it's a better idea to just use what is there, rather than taking to office hours or to WG.
* RG: A lot is already implemented — e.g., in CSS — and there is already usage. But in implementation, people found underspecified cases and need clearer requirements based on those things. We need to take another specification step to iron out those details.
```